app/src/main/java/com/example/sabonit/CartActivity.java

```diff
@@ -98,9 +100,23 @@ public void onProductDeleteClick(View view, int position) {
     }
 
     public void goToOrder(View view) {
-        showHowItWorksDialog();
```

Comment:
don't forget to make it work in the next version

Reply:
Yes, I'll work on it - when the dialog shows up the user can decide to continue to order or not